Source/Scene/DepthPlane.js

```diff
@@ -0,0 +1,185 @@
+/*global define*/
```

Comment:
Do we have enough tests for this?  Is it worth making a new file like [SkyBoxSpec.js](https://github.com/AnalyticalGraphicsInc/cesium/blob/master/Specs/Scene/SkyBoxSpec.js)?